app/views/bots/new.html.erb

```diff
@@ -1,32 +1,76 @@
-<br/>
-<% if current_developer %>
-  <div class="container">
-    <%= form_for @bot do |b| %>
-    Name:<%= b.text_field :name %>
-    <br/>
-    Username:<%= b.text_field :username %>
-    <br/>
-    Bio:<%= b.text_field :bio %>
-    <br/>
-    Tags:<%= b.text_field :tag_list %>
-    <br/>
-     <%= b.hidden_field :developer_id, value: current_developer.id %>
-    <br/>
-    Avatar:<%= b.file_field :avatar %>
-    <br/>
-    Cover:<%= b.file_field :cover %>
-    <br/>
-    <%= b.submit %>
-  <% end %>
-
-  <% if @bot.errors.any? %>
-    <ul class="Signup_Errors">
-      <% for message_error in @bot.errors.full_messages %>
-        <li>*
+<div class="card bg-light">
+  <article class="card-body mx-auto" style="max-width: 350px;">
+    <img class="card-img-top" src="<%= asset_path('logo.svg')%>" alt="", width="250", height="350">
+  <% if current_developer %>
+    <div class="container">
+      <div class="row h-100 justify-content-center align-items-center">
+        
+        <%= form_for @bot do |b| %>
+          <div class="form-group input-group">
+            <div class="input-group-prepend">
+              <span class="input-group-text">
+                <i class="fa fa-user"></i>
+              </span>
+              <%= b.text_field :name , placeholder: 'Name', class: 'form-control', type: 'Name', required: true%>
```

Comment:
None of the input fields are as wide as the file input field. They should have same width. 

![image](https://user-images.githubusercontent.com/51726526/114102129-269c5880-989d-11eb-88bc-b62e0b88123d.png)